src/liballoc_jemalloc/lib.rs

```diff
@@ -92,6 +98,16 @@ mod imp {
     }
 
     #[no_mangle]
+    pub extern "C" fn __rust_allocate_zeroed(size: usize, align: usize) -> *mut u8 {
+        if align <= MIN_ALIGN {
```

Comment:
I am not super familiar with jemalloc - is `calloc` faster than unconditionally calling `mallocx` here?